frontend/src/pages/Inspection/MetadataToolbar.jsx

```diff
@@ -0,0 +1,176 @@
+  height: 1.2rem;
+  color: #b47b38;
+  padding: 0.3rem;
+`;
+
+const FeedbackLink = styled(Link)`
+  display: inline-flex;
+  flex-direction: row;
+  align-items: center;
+  align-content: center;
+  background-color: #cee0ff;
+  border: solid 0.1rem #759ed5;
+  padding: 0;
+  border-radius: 2.0rem;
+  cursor: pointer;
+  margin: 0 0.5rem;
+  & > p {
+    font-size: 0.8rem;
+    text-align: center;
+    color: #6082af;
+    padding: 0 0.5rem 0 0.3rem;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
+  &:hover {
+    background-color: #deedff;
+  }
+`;
+const FeedbackIcon = styled(Book)`
+  height: 1.2rem;
+  color: #6082af;
+  padding: 0.3rem;
+`;
+
+const WriteFeedbackLink = styled(Link)`
+  margin-left: 0.5rem;
+  font-size: 0.8rem;
+`;
+
+const MetadataToolbar = ({ uuid, experiencePostGuid }) => {
+  const dispatch = useDispatch();
+  const [likes, setLikes] = useState(0);
+  const [isLiked, setLiked] = useState(false);
+  const userSelector = useSelector((state) => state.user) || { user: { token: null } };
+  const { user: token } = userSelector;
+
+  useState(() => {
+    const internal = async () => {
+      try {
+        const res = await fetch(`/api/Metadata/${uuid}/like`, {
+          method: 'GET',
+          headers: {
+            Authorization: `bearer ${token}`,
+          },
+        });
+        const { likeCount, liked } = await res.json();
+        setLikes(likeCount);
+        setLiked(liked);
+      } catch (err) {
+        const { status } = err;
+        if (status === 404) {
+          dispatch(alertActions.error('Could not fetch likes for dataset.'));
+        } else {
+          dispatch(alertActions.error('Failed to fetch likes. Please try again later.'));
+        }
+      }
+    };
+    internal();
+  }, []);
+
+  const handleLike = async () => {
+    try {
+      const res = await fetch(`/api/Metadata/${uuid}/like`, {
+        method: 'PUT',
+        headers: {
+          Authorization: `bearer ${token}`,
+        },
+      });
+      const { status } = res;
+      if (status === 200) {
+        setLiked(!isLiked);
+        setLikes(isLiked ? likes - 1 : likes + 1);
+      }
+    } catch (err) {
+      dispatch(alertActions.error('Something went wrong when adding/removing star.'));
+    }
+  };
+
+  return (
+    <MetadataToolbarContainer>
+      <FavouriteButton onClick={handleLike}>
+        {isLiked ? <StarFullStyled /> : <StarEmptyStyled />}
+        <p>Interesting</p>
+        <LikeCounter>{likes}</LikeCounter>
+      </FavouriteButton>
+      <div>
+        {
+          experiencePostGuid
+            ? (
+              <FeedbackLink to={`/articles/${experiencePostGuid}`}>
+                <FeedbackIcon />
+                <p>Experience Article</p>
+              </FeedbackLink>
+            )
+            : <WriteFeedbackLink to={`/articles/new/${uuid}`}>Are you the owner of this data set? Write an experience article here</WriteFeedbackLink>
```

Comment:
<img width="572" alt="Skjermbilde 2020-03-05 kl  17 05 59" src="https://user-images.githubusercontent.com/15128509/76000278-c6055400-5f03-11ea-986c-63360470693f.png">
This isnt centered

Reply:
It wasn't supposed to be :|

Reply:
smh, always center your elements

Reply:
On it